rust-bins/src/bin/identity_disclosure.rs

```diff
@@ -73,63 +73,70 @@ struct Combine {
+    #[structopt(long = "out", help = "File to output the public holder identifier to.")]
     out:        PathBuf,
 }
 
 #[derive(StructOpt)]
-struct ComputeRegIds {
-    #[structopt(long = "ar-record", help = "The anonymity revocation record.")]
-    ar_record:      PathBuf,
-    #[structopt(long = "prf-key", help = "File containing the PRF key.")]
-    prf_key:        PathBuf,
+struct ComputeCredIds {
+    #[structopt(long = "id-record", help = "The identity record.")]
+    id_record:      PathBuf,
+    #[structopt(long = "key", help = "File containing the decrypted linking key.")]
+    key:            PathBuf,
     #[structopt(long = "global-context", help = "File with global context.")]
     global_context: PathBuf,
-    #[structopt(long = "out", help = "File to output the RegIds to")]
+    #[structopt(
+        long = "out",
+        help = "File to output the account credential identifiers to"
+    )]
     out:            PathBuf,
     #[structopt(
         long = "no-secret",
-        help = "Do __not__ output the decryption key together with the RegId."
+        help = "Do __not__ output the account decryption key together with the account credential \
+                identifiers."
     )]
     no_secret:      bool,
 }
 
 #[derive(StructOpt)]
 #[structopt(
-    about = "Prototype tool showcasing anonymity revoker (inter)actions.",
+    about = "Prototype tool showcasing identity disclosure (inter)actions.",
     author = "Concordium",
-    version = "0.5"
+    version = "0.6"
 )]
-enum AnonymityRevocation {
+enum IdentityDisclosure {
     #[structopt(
         name = "decrypt",
-        about = "Take a deployed credential and decrypt a share of idCredPub."
+        about = "Take a deployed account credential and decrypt a share of the public holder \
+                 identifier."
     )]
     Decrypt(Decrypt),
     #[structopt(
         name = "combine",
-        about = "Combine decrypted shares of anonymity revokers to get idCredPub."
+        about = "Combine decrypted shares of privacy guardians to get the public holder \
+                 identifier."
     )]
     Combine(Combine),
     #[structopt(
-        name = "decrypt-prf",
-        about = "Take an anonymity revocation record and let one anonymity revoker decrypt its \
-                 share of the PRF key."
+        name = "decrypt-linking-key",
+        about = "Take an identity record and let one privacy guardian decrypt its share of the \
+                 linking key."
     )]
-    DecryptPrf(DecryptPrf),
+    DecryptLinkingKey(DecryptLinkingKey),
     #[structopt(
-        name = "combine-prf",
-        about = "Combine decrypted shares of the PRF key to reconstruct the PRF key."
+        name = "combine-linking-key",
+        about = "Combine decrypted shares of the linking key to reconstruct the linking key."
     )]
-    CombinePrf(CombinePrf),
+    CombineLinkingKey(CombineLinkingKey),
     #[structopt(
         name = "compute-regids",
```

Comment:
If renaming regid->credid, probably also do it here

Reply:
Make sense. I have updated the code.